src/Compilers/Core/Portable/CommandLine/AnalyzerConfig.cs

```diff
@@ -115,9 +115,9 @@ public static AnalyzerConfig Parse(string text, string pathToFile)
         /// Parses an editor config file text located at the given path. No parsing
         /// errors are reported. If any line contains a parse error, it is dropped.
         /// </summary>
-        public static AnalyzerConfig Parse(SourceText text, string pathToFile)
+        public static AnalyzerConfig Parse(SourceText text, string? pathToFile)
         {
-            if (!Path.IsPathRooted(pathToFile) || string.IsNullOrEmpty(Path.GetFileName(pathToFile)))
+            if (pathToFile is null || !Path.IsPathRooted(pathToFile) || string.IsNullOrEmpty(Path.GetFileName(pathToFile)))
```

Comment:
`IsPathRooted` already handles `null` but that that doesn't help the analysis below hence added an explicit `null` check.

Reply:
It should. [https://github.com/dotnet/runtime/commit/25ea6c44cf3140cc9bfbaaff7f01ad2a27414f8e](https://github.com/dotnet/runtime/commit/25ea6c44cf3140cc9bfbaaff7f01ad2a27414f8e) was over a year ago.

---
In reply to: [502078334](https://github.com/dotnet/roslyn/pull/48460#discussion_r502078334) [](ancestors = 502078334)

Reply:
Nearly a year ago but not in netcoreapp3.1